app/src/main/java/com/dankideacentral/dic/ConstantValues.java

```diff
@@ -0,0 +1,20 @@
+package com.dankideacentral.dic;
+
+/**
+ * Created by basim on 10/6/16.
+ * This file was part of the tutorial I was using. 
+ *
+ * TODO: Use SharedPreferences instead of this shitty file.
+ */
+
+public class ConstantValues {
```

Comment:
Instead of a ConstantValues class, I believe it would be considered better practice to have resources in the `strings.xml` file.